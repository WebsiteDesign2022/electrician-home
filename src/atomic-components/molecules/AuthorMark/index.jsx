import './index.scss';

const AuthorMark = ({ text }) => (<div className="text-white">{text}</div>);

export default AuthorMark;
