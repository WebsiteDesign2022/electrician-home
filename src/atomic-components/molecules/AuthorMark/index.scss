.author-mark {
    padding-left: 16px;
}