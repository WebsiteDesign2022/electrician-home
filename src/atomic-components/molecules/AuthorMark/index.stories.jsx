import React from 'react';

import AuthorMark from '.';

export default {
  title: 'Electrician-Home/molecules/AuthorMark',
  component: AuthorMark,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <AuthorMark {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
