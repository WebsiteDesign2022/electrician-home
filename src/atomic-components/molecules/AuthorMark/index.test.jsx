import TestRenderer from 'react-test-renderer';

import AuthorMark from '.';

describe("AuthorMark", () => {
    test("renders AuthorMark", () => {
        const authorMark = TestRenderer.create(
            <AuthorMark text="author-mark-text" />
        );

        expect(authorMark.toJSON()).toMatchSnapshot();
    });
});
