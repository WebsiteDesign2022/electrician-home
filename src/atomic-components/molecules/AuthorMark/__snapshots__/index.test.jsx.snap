// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorMark renders AuthorMark 1`] = `
<div
  className="text-white"
>
  author-mark-text
</div>
`;
